packages/wonder-blocks-dropdown/src/components/dropdown-opener.tsx

```diff
@@ -51,15 +51,21 @@ type Props = Partial<Omit<AriaProps, "aria-disabled">> & {
      * If the dropdown has an error.
      */
     error?: boolean;
+    /**
+     * The role of the opener.
+     */
+    role: "combobox" | "button";
 };
 
 type DefaultProps = {
     disabled: Props["disabled"];
+    role: Props["role"];
```

Comment:
@beaesguerra or @kevinb-khan out of curiosity, do you know why this object access pattern is used for DefaultProps?